### ISSUE TITLE
Error popup with no explanation and no online references

### ISSUE THREAD

**Issue by jghodd** [ibus/ibus#2306]
Please fill in the following items if you don't know the root cause.

Which distribution and version?: 
Arch 5.11.1

Which desktop environment and version?: 
KDE/Plasma

Which session type?: 
X11

Which application and version?: 
On desktop startup

IBus version?: 
1.5.24

Issue description:
After upgrading from v1.5.23 to 1.5.24, I'm getting a popup message that reads:

"At least one of your configured input methods does not exist in
IBus input methods. Please run 'ibus-setup' command, open "Input
Method" tab, and configure your input methods again."

I can find no reference to this anywhere, and no explanation of why this popup suddenly started coming up.

Steps to reproduce:
1. Reboot system
2. Log in to desktop
 

Can you reproduce your problem when you restart ibus-daemon? (yes / no): 
No. After 'ibus exit' I can no longer enter anything on the command line.

Do you see any errors when you run ibus-daemon with the verbose option?:
No

Can you reproduce your problem with a new user account instead of the current your account? (yes / no): 
No

**Comment by jghodd** [comment]
I've determined that adding an input method, any input method, causes the popup not to appear on next boot/login. So, it would seem that adding any input method is causing *something* to be initialised that used to be initialised in v1.5.23, but is not being initialised in v1.5.24 without adding an input method, despite the fact that about a dozen or more input methods already exist.

What is causing the error popup to appear, why does one have to add a random input method to properly initialise ibus and what needs to be done to fix this?

**Comment by jghodd** [comment]
I have been able to generate a fixed .config/dconf/user file by adding the English(UK) input method using my live system iso image and saving off the modified user file. I'd have preferred if someone could have responded and provided some answers as to what may or may not be the issue. I consider this a workaround, not a fix. Something changed in v1.5.24 and whatever that was, it broke ibus - there's a bug somewhere.

Nonetheless, at this point I assume I'm not going to get any assistance, so I'm closing the issue.

**Comment by eclairevoyant** [comment]
Shame no one responded to you, I ended encountering the same issue recently.
I figured it was something to do with ibus's user config, which according to https://unix.stackexchange.com/a/236817 is handled by dconf. It's possible that (for example) you had an input method added, uninstalled it, and it stayed in the config even after uninstallation, and failed to load on restart - or for whatever other reason the config may be invalid.

As mentioned at the link above, `dconf dump /desktop/ibus/` should tell you what the current config is and that would be a good starting point for diagnosing further.
